packages/grafana-data/src/types/explore.ts

```diff
@@ -27,3 +27,17 @@ export interface ExploreTracePanelState {
 export type SplitOpen = <T extends DataQuery = any>(
   options?: { datasourceUid: string; query: T; range?: TimeRange; panelsState?: ExplorePanelsState } | undefined
 ) => void;
+
+export type ExplorePanelProps = {
```

Comment:
this should be streamlined and some props may no really need to be passed but there will probably always be differences between Explore and Dashboard props.